<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editionBonCommandes</class>
 <widget class="QWidget" name="editionBonCommandes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition bon de commande</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/BonCommandes.jpg</normaloff>:/images/BonCommandes.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style pour les QPushButton */
QPushButton {
    background-color: #4CAF50;  /* Vert */
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 10px;
    padding: 10px;
	font-weight: bold;
	font-size: 15px
}

QPushButton:hover {
    background-color: #45a049;  /* Vert plus foncé */
}

QPushButton:pressed {
    background-color: #388E3C;  /* Vert encore plus foncé */
}

/* Style pour les QLineEdit */
QLineEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

QComboBox{
	background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDateEdit */
QDateEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QSpinBox */
QSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDoubleSpinBox */
QDoubleSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour le QTableView */
QTableView {
    background-color: white;
    gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;

}
QTableView:pressed {
	bacground-color:rgb(188, 188, 188);
	 gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;
}


QHeaderView::section {
    background-color: #4CAF50;
    color: white;
    padding: 4px;
    border: 1px solid #4CAF50;
}

/* Style pour les QLabel */
QLabel {
    color: black;
    font-weight: bold;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Accueil">
       <property name="styleSheet">
        <string notr="true">background-color:white;
color:black;</string>
       </property>
       <property name="text">
        <string>Acceuil</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tresorerie">
       <property name="styleSheet">
        <string notr="true">background-color:white;
color:black;</string>
       </property>
       <property name="text">
        <string>Trésorerie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rapport_statistique">
       <property name="styleSheet">
        <string notr="true">background-color:white;
color:black;</string>
       </property>
       <property name="text">
        <string>Rapport et statistique</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>628</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="styleSheet">
      <string notr="true">background-color: green;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Edition  du bon de commande&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Fournisseur</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="fournisseur_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Quantité</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="qte_spinBox">
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="description_lineEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Prix U.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="prixUnitaire_doubleSpinBox">
           <property name="maximum">
            <double>10000000000000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Total</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="total_doubleSpinBox">
           <property name="maximum">
            <double>999999999999999945575230987042816.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="codeproduit_lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Code produit</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="nbrePieces_doubleSpinBox">
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Nbre de Pièces</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modify_pushButton">
           <property name="styleSheet">
            <string notr="true">background-color:blue;
color:white;</string>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="styleSheet">
            <string notr="true">background-color:white;
color:black;</string>
           </property>
           <property name="text">
            <string>Annuler</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView_2">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="search_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;</string>
         </property>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#00aa00;&quot;&gt;PharmaStock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Arafat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>118 Parakou</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>+229 61359067</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>pharmastock@gmail.fr</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>128</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" rowspan="2">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Commande N° :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="numeroCommandeLabel">
           <property name="text">
            <string>00 XXX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="DateLabel">
           <property name="text">
            <string>15/03/2024</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Fournisseur :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="NomFournisseurLabel">
           <property name="text">
            <string>Ninho Laboratoire</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Contact</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="AdresseFournisseurLabel">
           <property name="text">
            <string>USA</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="ContactFournisseurLabel">
           <property name="text">
            <string>+1 45&quot;689876689</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="delete_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color:red;
color:white;</string>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="valider_pushButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="print_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color:blue;
color:white;</string>
         </property>
         <property name="text">
          <string>Imprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="validerEtImprimer_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color:blue;
color:white;</string>
         </property>
         <property name="text">
          <string>Valider et imprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label_16</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder></zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
